packages/devtools_app/lib/src/network/network_request_inspector_views.dart

```diff
@@ -151,6 +160,92 @@ class HttpResponseView extends StatelessWidget {
   }
 }
 
+class ImageResponseView extends StatelessWidget {
+  const ImageResponseView(this.data);
+
+  final DartIOHttpRequestData data;
+
+  @override
+  Widget build(BuildContext context) {
+    final img = image.decodeImage(data.encodedResponse);
+    // Format should be of the form: [image/$format]
```

Comment:
can you use quotes in this comment instead? I was (may still be) confused on whether the brackets are part of the expected form value

Reply:
The brackets were part of the expected format (e.g., "[image/$format]"), but I've just removed it.